bin/oneoff/data_fix/bad_school_info_in_users

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+UPDATING = (ARGV[0].eql? "true").freeze
+puts "Called with UPDATING=#{UPDATING}"
```

Comment:
Maybe worth adding a little blurb about what it means and what the valid options are.

It also seems a bit odd to me that it will update if the argument is `true`. 
e.g. 

```
> bad_school_info_in_users
  Called with UPDATING=false

> bad_school_info_in_users nonsense
  Called with UPDATING=false

> bad_school_info_in_users true
  Called with UPDATING=true
```

Maybe require the argument to actually update be something like `ACTUALLY_UPDATE` instead of just `true`

Reply:
Do you mean here just renaming the variable or supplying the argument as `--actually-update true` ?

Reply:
Yes, though I don't think we need the `true` part. It can just be `--actually-update` with or without the `--` prefix. I just like the argument, especially here since it's causing the script to be destructive, to be more self-explanatory.